src/frontend/Komponenter/Behandling/Brev/BrevInnhold.tsx

```diff
@@ -196,17 +194,23 @@ const BrevInnhold: React.FC<Props> = ({
                                 </Innholdsrad>
                                 <FlyttAvsnittKnappWrapper id={knappWrapperId}>
                                     {index > 0 && (
-                                        <OppKnapp
+                                        <Button
+                                            type={'button'}
                                             onClick={() => {
                                                 flyttAvsnittOpp(rad.id);
                                             }}
+                                            variant={'tertiary'}
+                                            icon={<Up />}
                                         />
                                     )}
                                     {index + 1 < avsnittSomSkalVises.length && (
-                                        <NedKnapp
+                                        <Button
```

Comment:
Hmm, hade kanskje ønsket vi kunde bruke egen-definierte knapper mer, for lesbarhet 👀 Blir mye boilerplate med type variant og icon

Reply:
Enig med Johan her - det fine med "oppknapp" / "nedknapp" er at det er "lett" å endre alle på ett sted dersom det trengs. Det er slik jeg ser det "gode" abstraksjonskomponenter for vårt bruk.

Reply:
Fixd [eda9b1e](https://github.com/navikt/familie-ef-sak-frontend/pull/1767/commits/eda9b1e93c7c5ce45b11bb036bfeebe8eb0bd8b4)